src/web/badge/src/lib.rs

```diff
@@ -34,12 +35,11 @@ pub struct Badge {
 
 impl Badge {
     pub fn new(options: BadgeOptions) -> Result<Badge, String> {
-        let collection = FontCollection::from_bytes(FONT_DATA).expect("Failed to parse FONT_DATA");
-
-        // this should never fail in practice
-        let font = collection
-            .into_font()
-            .map_err(|_| "Failed to load font data".to_owned())?;
+        static FONT: OnceCell<Font> = OnceCell::new();
+        // Font is an `Arc` and therefore cheap to clone
+        let font = FONT.get_or_init(|| {
+            Font::try_from_bytes(FONT_DATA).expect("Failed to parse FONT_DATA")
+        }).clone();
```

Comment:
I'd rather use a `Lazy`
```suggestion
static FONT: Lazy<Font> = Lazy::new(|| {
    Font::try_from_bytes(FONT_DATA).expect("Failed to parse FONT_DATA")
});
let font = FONT.clone();
```
I don't like the clone and kinda feel like that can be removed, but that's a separate issue

Reply:
Ooh, lazy is a great idea.

The clone here is really cheap because all the font variants are `Arc`s: https://docs.rs/rusttype/0.9.2/rusttype/enum.Font.html